src/classes/app.py

```diff
@@ -232,3 +246,22 @@ def run(self):
 
         # return exit result
         return res
+
+    # Log the session's end
+    @pyqtSlot()
+    def onLogTheEnd(self):
+        """ Log when the primary Qt event loop ends """
+
+        try:
+            from classes.logger import log
+            import time
+            now = time.asctime()
+            new_session = 'OpenShot\'s session ended'
+            log.info("         " + new_session.center(30))
+            log.info("         " + now.center(30))
+            log.info("================================================")
+        except:
+            pass
```

Comment:
[No (more) bare `except:`s](https://moserei.de/2017/09/20/python-bare-except.html), please. The code is riddled with enough that I've been meaning to get rid of.

`except Exception as ex:`, or `except Exception:` if you don't need access to `ex` (like here).